hardware_interface/include/hardware_interface/types/hardware_interface_type_values.hpp

```diff
@@ -25,6 +25,28 @@ constexpr char HW_IF_VELOCITY[] = "velocity";
 constexpr char HW_IF_ACCELERATION[] = "acceleration";
 /// Constant defining effort interface name
 constexpr char HW_IF_EFFORT[] = "effort";
```

Comment:
Let's remove effort and use explicitly `torque` and `force`.

Reply:
Are we going to do it with Jazzy? or in future?